pkg/cloud/openstack/cluster/actuator.go

```diff
@@ -66,12 +67,20 @@ func (a *Actuator) Reconcile(cluster *clusterv1.Cluster) error {
 		return err
 	}
 
+	certificatesService := certificates.NewService()
+
 	// Load provider spec & status.
 	clusterProviderSpec, clusterProviderStatus, err := providerv1.ClusterSpecAndStatusFromProviderSpec(cluster)
```

Comment:
ClusterSpecAndStatusFromProviderSpec    is this implemented?

Reply:
Yup here: https://github.com/sbueringer/cluster-api-provider-openstack/blob/pr-implement-reconcile-certs/pkg/apis/openstackproviderconfig/v1alpha1/register.go#L41

If it's not in the diff, it's because I implemented it in the PR before.

Reply:
ok, got it, thanks ~